Sources/StreamChat/Database/DatabaseContainer.swift

```diff
@@ -289,6 +289,7 @@ class DatabaseContainer: NSPersistentContainer {
         allContext.forEach { context in
             context.perform {
                 context.invalidateCurrentUserCache()
+                context.reset()
```

Comment:
hmm, this doesn't seem like it should be fixed here, but in `develop`. @nuno-vieira  thoughts?